packages/mdc-slider/README.md

```diff
@@ -0,0 +1,166 @@
+### JavaScript Instantiation
+
+```js
+import {MDCSlider} from '@material/slider';
+
+const slider = new MDCSlider(document.querySelector('.mdc-slider'));
+```
+
+## Variants
+
+### Continuous Slider
+
+Use continuous sliders allow users to make meaningful selections that don’t require a specific value. Continuous Slider is the default variant.
+
+### Initializing the slider with custom ranges/values
+
+When `MDCSlider` is initialized, it reads the element's `aria-valuemin`, `aria-valuemax`, and
+`aria-valuenow` values if present and uses them to set the component's `min`, `max`, and `value`
+properties. This means you can use these attributes to set these values for the slider within the
+DOM.
+
+```html
+<div class="mdc-slider" tabindex="0" role="slider"
+     aria-valuemin="-5" aria-valuemax="50" aria-valuenow="10"
+     aria-label="Select Value">
+  <!-- ... -->
+</div>
+```
+
+## Style Customization
+
+### CSS Classes
+
+CSS Class | Description
+--- | ---
+`mdc-slider` | Mandatory.
+`mdc-slider__track` | Mandatory. Element containing the track-fill.
+`mdc-slider__track-fill` | Mandatory. The fill element to display where the value is.
+`mdc-slider__thumb` | Mandatory. Element containing the thumb-handle.
+`mdc-slider__thumb-handle` | Mandatory. The handle element to display where the value is.
+
+## `MDCSlider` Properties and Methods
+
+| Property | Value Type | Description |
+| --- | --- | --- |
+| `value` | `number` | The current value of the slider. Changing this will update the slider's value. |
+| `min` | `number` | The minimum value a slider can have. Values set programmatically will be clamped to this minimum value. Changing this property will update the slider's value if it is lower than the new minimum |
+| `max` | `number` | The maximum value a slider can have. Values set programmatically will be clamped to this maximum value. Changing this property will update the slider's value if it is greater than the new maximum |
+
+| Method Signature | Description |
+| --- | --- |
+| `layout() => void` | Recomputes the dimensions and re-lays out the component. This should be called if the dimensions of the slider itself or any of its parent elements change programmatically (it is called automatically on resize). |
+
+### Additional Information
+
+#### Events
+
+`MDCSlider` emits a `MDCSlider:input` custom event from its root element whenever the slider value
+is changed by way of a user event, e.g. when a user is dragging the slider or changing the value
+using the arrow keys. The `detail` property of the event is set to the slider instance that was
+affected.
+
+`MDCSlider` emits a `MDCSlider:change` custom event from its root element whenever the slider value
+is changed _and committed_ by way of a user event, e.g. when a user stops dragging the slider or
+changes the value using the arrow keys. The `detail` property of the event is set to the slider
+instance that was affected.
+
+## Usage within Web Frameworks
+
+#### `MDCSliderAdapter`
```

Comment:
nit: Use level 3 heading (and same with `MDCSliderFoundation` below)

Reply:
+1